attributes/default.rb

```diff
@@ -6,12 +6,13 @@
 default['amazon-ecs-agent']['tag'] = 'latest'
 default['amazon-ecs-agent']['user'] = 'ubuntu'
 default['amazon-ecs-agent']['storage_driver'] = 'overlay'
-default['amazon-ecs-agent']['docker']['version'] = '1.11.2'
+default['amazon-ecs-agent']['docker']['version'] = '17.05.0'
 default['amazon-ecs-agent']['docker']['apt']['keyserver'] = 'hkp://p80.pool.sks-keyservers.net:80'
-default['amazon-ecs-agent']['docker']['apt']['key'] = '58118E89F3A912897C070ADBF76221572C52609D'
+default['amazon-ecs-agent']['docker']['apt']['key'] = '0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e'
+default['amazon-ecs-agent']['docker']['apt']['uri'] = 'https://packages.docker.com/1.13/apt/repo'
```

Comment:
Nah let's get rid of them. We might have to change the whole command and not just the URL next time it changes, so we may as well change the recipe at that time.